dist/sass/_generated/_icon.scss

```diff
@@ -243,23 +291,26 @@
 @mixin dc-icon-generated-selectors {
     @font-face {
         font-family: "DressCodeIcons";
```

Comment:
@gregdeane It seem that here the cache busting string ```?v=2.0.0-beta``` is not getting the right version